src/apphosting/config.ts

```diff
@@ -1,8 +1,9 @@
 import * as path from "path";
 import { writeFileSync } from "fs";
-import * as yaml from "js-yaml";
+import * as yaml from "yaml";
 
 import * as fs from "../fsutils";
+import { NodeType } from "yaml/dist/nodes/Node";
```

Comment:
This seems like an internal type for the library, and they explicitly don't use it in their public API. https://github.com/eemeli/yaml/blob/main/src/public-api.ts

Do you need this type? Could this be refactored not to use it?

Reply:
I think this is an oversight for it not to be exported. AFAICT none of the types are exported (note in public-api, Node is imported, and is a mandatory base class of Document<T, Strict> but is not exported in that file either.

Frankly, the type annotations are a mess; I had it working and removed it all.  I could hypothetically remove the API call, but it would instead involve yet more casting as any/unknown.